io.github.monsterbirdlimited.soscap.yaml

```diff
@@ -0,0 +1,81 @@
+---
+app-id: io.github.monsterbirdlimited.soscap
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: soscap
+finish-args:
+  - --filesystem=home
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --device=dri
+  - --talk-name=org.freedesktop.FileManager1
+  - --filesystem=xdg-videos
+  - --env=G_MESSAGES_DEBUG=none
+  - --env=GST_DEBUG=3
+
+
+cleanup:
+  - '/include'
+  - '/lib/pkgconfig'
+  - '/man'
+  - '/share/doc'
+  - '/share/gtk-doc'
+  - '/share/man'
+  - '/share/pkgconfig'
+  - '*.la'
+  - '*.a'
+
+modules:
+  
+  - name: soscap
+    buildsystem: simple
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/Monsterbirdlimited/soscap
+        commit: c58ffa8d785d090a7191a0467621cc56ec9b7222
+        tag: v0.9.1-alpha 
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\\d.]+)$
+          is-main-source: true
+    build-commands:
+      - install -Dm755 soscap.py ${FLATPAK_DEST}/bin/soscap
+      - install -Dm644 io.github.monsterbirdlimited.soscap.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 io.github.monsterbirdlimited.soscap.png ${FLATPAK_DEST}/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.png
+
+  - name: x264
+    config-opts:
+      - --enable-shared
+      - --disable-cli
+      - --system-libx264
+      - --enable-pic
+      - --enable-lto
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        branch: stable
+
+  - name: libx264
+    config-opts:
+      - --enable-shared
+      - --disable-cli
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        branch: stable
```

Comment:
never a branch. You need a specific commit or tag.

Reply:
okay. i will try to get it and make the changes